pom.xml

```diff
@@ -106,6 +107,11 @@
       <artifactId>guava</artifactId>
       <version>19.0</version>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
```

Comment:
Why not use `commons-lang3` instead?
Also, unless I'm misreading the changes Github is displaying here, this newly introduced dependency is not actually used.

Reply:
The non-obvious reason is that previously we where using class loader of base plugin and commons-lang was available. We have some classes which use this lib (not changed/added in this commit) like RulesLoader or HttpConstantCheck.

Reply:
Yepp, agree. This is basically carry-over, but since we have full control now, maybe upgrade of this library is worth exploring.

Reply:
Question is do we want to do this in this issue?

Reply:
If you change it and then feel comfortable with that, why not. Hopefully, it will only change imports.